my.exception=it's a global exception.